src/pcapi/flask_app.py

```diff
@@ -91,6 +95,13 @@
 
 @app.before_request
 def before_request() -> None:
+    if current_user and current_user.is_authenticated:
+        set_user(
```

Comment:
Le lecteur pourrait se demander d'où vient`set_user()`. Suggestion : 
```suggestion
        sentry_sdk.set_user(
```

Idem pour `set_tag()`.